src/functions.rs

```diff
@@ -1304,11 +1304,12 @@ pub struct FunctionInfo<'a> {
     pub is_virtual: bool,
     pub args: Vec<FunctionArgument<'a>>,
     pub annotations: BTreeSet<AnnotationInfo>,
-    original_body: &'a [Statement],
+    pub original_body: &'a [Statement],
     pub body: Option<BTreeSet<ValidatedStatement<'a>>>,
     pub declaration_file: &'a SimpleFile<String, String>,
     pub is_associated_call: bool,
     pub is_derived: bool,
+    pub is_castable: bool,
```

Comment:
Does it compile if we make this private?

Either way, can we add a comment that says that this should typically be initialized to true, and prevalidate_functions() will set it to false if needed?

Reply:
Private breaks things, but I did add the comment.